code/game/objects/effects/grubs_vr.dm

```diff
@@ -0,0 +1,127 @@
+/obj/effect/spider/grub //travis is a butt!!!

```

Comment:
You're inheriting the Spider effect but appear to have mostly just copied the Spider effect. I don't understand.

Reply:
Will look at this, see if I can cut down the code directly copied

Reply:
going to look into this.